pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -358,6 +370,86 @@ func (r *ReconcileVirtualMachineImport) upsertStatusConditions(vmiName types.Nam
 	return nil
 }
 
+func (r *ReconcileVirtualMachineImport) storeSourceVMStatus(instance *v2vv1alpha1.VirtualMachineImport, vmStatus string) error {
+	vmiCopy := instance.DeepCopy()
+	vmiCopy.Annotations[sourceVMInitialState] = vmStatus
```

Comment:
This is failing on:
```
panic: assignment to entry in nil map
```
we need to init the map before using it.

Reply:
Interesting... For some reason, in my deployment it wasn't empty. Good catch!